Kickstarter-iOS/Views/Cells/ProjectPamphletMainCell.swift

```diff
@@ -34,6 +34,10 @@ internal final class ProjectPamphletMainCell: UITableViewCell, ValueCell {
       |> \.translatesAutoresizingMaskIntoConstraints .~ false
   }()
 
+  private lazy var tapGesture: UITapGestureRecognizer = {
```

Comment:
Could we call this `creatorBylineTapGesture` just to make it easier to know what this is in future.